plugins/tekton/src/components/PipelineRunLogs/PipelineRunLogViewer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { DismissableBanner, LogViewer } from '@backstage/core-components';
+
+import { V1Container, V1Pod } from '@kubernetes/client-node';
+import { Paper } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { upperCase } from 'lodash';
```

Comment:
we can avoid `lodash` and use built in `toUpperCase`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase